src/main/java/chess/view/InputView.java

```diff
@@ -0,0 +1,35 @@
+package chess.view;
+
+import chess.domain.Command;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    public String readInitCommand() {
+        System.out.println("> 체스 게임을 시작합니다." + System.lineSeparator()
+                + "> 게임 시작 : start" + System.lineSeparator()
+                + "> 게임 종료 : end" + System.lineSeparator()
+                + "> 게임 이동 : move source위치 target위치 - 예. move b2 b3");
```

Comment:
자바 15부터는 여러 라인의 문자열을 쉽게 사용할 수 있는 "Multi-line String" 기능을 제공합니다. 
```suggestion
        System.out.println("""
                > 체스 게임을 시작합니다.
                > 게임 시작 : start
                > 게임 종료 : end
                > 게임 이동 : move source위치 target위치 - 예. move b2 b3""");
```